TSC/hosted-project-governance.md

```diff
@@ -0,0 +1,29 @@
+# Hosted Project Governance
+Published: March 10, 2025
+
+## Definition
+The Bytecode Alliance Technical Steering Committee (TSC) [charter](https://github.com/bytecodealliance/governance/blob/main/TSC/charter.md) defines two types of Bytecode Alliance Projects -- "Hosted" and "Core", saying:
+> By default a project adopted by the BA is considered a Hosted Project. The BA board designates specific projects at Core Projects, requiring adherence to heightened standards around project governance, mission alignment, communication, and security. … Core Projects must meet all the requirements of Hosted Projects, but each project has exactly one status – Hosted or Core.
```

Comment:
```suggestion
> By default a project adopted by the BA is considered a Hosted Project. The BA board also designates specific projects as Core Projects, requiring adherence to heightened standards around project governance, mission alignment, communication, and security. Core Projects must meet all the requirements of Hosted Projects, but each project has exactly one status – Hosted or Core.
```

Reply:
Edits incorporated, thank you.  This wording was originally copied/pasted directly from the TSC charter but it makes sense to adjust it here as proposed.